src/main/scala/io/keen/client/scala/Client.scala

```diff
@@ -361,6 +368,26 @@ trait Writer extends AccessLevel {
    */
   val writeKey: String = settings.writeKey.getOrElse(throw MissingCredential("Write key required for Writer"))
 
+  // constants
+  protected val MinSendIntervalEvents: Long = 100
+  protected val MaxSendIntervalEvents: Long = 10000
+  protected val MinSendIntervalSeconds: Long = 60
+  protected val MaxSendIntervalSeconds: Long = 3600
+
+  // initialize and configure our local event store queue
+  val batchSize: Integer = settings.batchSize.getOrElse(500)
+  val batchTimeout: Integer = settings.batchTimeout.getOrElse(5)
+  val eventStore: EventStore = new RamEventStore
+  eventStore.maxEventsPerCollection = settings.maxEventsPerCollection.getOrElse(10000)
```

Comment:
This seems like it should be a constructor argument rather than set independently.